### PR TITLE
[NOTEPAD] Update Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**
Attendum to https://github.com/reactos/reactos/pull/6517

I fixed the entire translation and all the accelerators. Now everything is like in MSDN.

### CODE REVIEW

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian

**Review by JoachimHenze — APPROVED** [review]

File: base/applications/notepad/lang/ro-RO.rc [hunk 1]
```diff
@@ -33,46 +33,46 @@ MAIN_MENU MENU
...
+        MENUITEM "&Selectare totală\tCtrl+A", CMD_SELECT_ALL
+        MENUITEM "Dată/&Oră\tF5", CMD_TIME_DATE
     END
-    POPUP "F&ormatare"
+    POPUP "F&ormat"
     BEGIN
-        MENUITEM "În&cadrează liniile lungi", CMD_WRAP
-        MENUITEM "&Font…", CMD_FONT
+        MENUITEM "&Încadrare cuvânt", CMD_WRAP
+        MENUITEM "&Font...", CMD_FONT
```
Comment (JoachimHenze) [inline base/applications/notepad/lang/ro-RO.rc:67]: Why the change from "&Font…" to "&Font..." ? (1char to 3chars for the ...) The former is more compact horizontally, therefore is a good default choice (or whenever fearing that space is limited).
Same question for other places where you did that change.

Comment (AndreiMiloiu) [reply base/applications/notepad/lang/ro-RO.rc:67]: I saw thatg in English there are three dots as three characters, too. The same in French, Hungarian, German and many other resources.

Comment (JoachimHenze) [reply base/applications/notepad/lang/ro-RO.rc:67]: Like I said, I do think the single-character-"…" is a sane choice whenever we are in a rc file which already contains multibyte-characters, which means typically almost all rcs except en-US.
Yes, MS doesn't do that, but I see no urgent need to match MS exactly in that specific detail.
Often the single-character-"…" allowed us to make a line fit to a rather narrow button. So changing an existing single-character-"…" to the regular 3 dots is something that I would generally discourage.

Yes, my personal recommendation is to stick with the single-character-"…" !

File: base/applications/notepad/lang/ro-RO.rc [hunk 2]
```diff
@@ -161,18 +161,18 @@ BEGIN
...
-Încercați un alt editor."
-    STRING_NOTEXT "Nu ați scris nimic.\
-\nScrieți ceva apoi încercați din nou."
-    STRING_DOESNOTEXIST "Fișierul „%s”\nnu există.\n\n\
-Creați un nou fișier?"
+    STRING_TOOLARGE "Fișierul „%s” este prea lung pentru Notepad...\n\
+Utilizaţi alt editor pentru acest fişier."
+    STRING_NOTEXT "Nu ați scris nimic.\n\
+Scrieți ceva apoi încercați din nou."
+    STRING_DOESNOTEXIST "Fișierul „%s”\n\nu există.\n\n\
```
Comment (JoachimHenze) [inline base/applications/notepad/lang/ro-RO.rc:168]: you changed the text from 1 newline plus "nu există"
to 2 newlines plus "u există".
Was that intentional or a typo?

File: base/applications/notepad/lang/ro-RO.rc [hunk 3]
```diff
@@ -33,46 +33,46 @@ MAIN_MENU MENU
...
-    POPUP "&Afișare"
+    POPUP "&Vizualizare"
     BEGIN
         MENUITEM "&Bară de stare", CMD_STATUSBAR
     END
-    POPUP "Aj&utor"
+    POPUP "&Ajutor"
     BEGIN
-        MENUITEM "&Vizualizare Ajutor…", CMD_HELP_CONTENTS
+        MENUITEM "&Termeni din Ajutor", CMD_HELP_CONTENTS
```
Comment (JoachimHenze) [inline base/applications/notepad/lang/ro-RO.rc:75]: This lines change is okay. My (german) WinXPSP3 notepad also doesn't have any dots here in the end.

File: base/applications/notepad/lang/ro-RO.rc (hunk 2 above)
Comment (AndreiMiloiu) [reply base/applications/notepad/lang/ro-RO.rc:168]: Bht \n\ doesn't mean a new line? I didn't want to do any typo.

File: base/applications/notepad/lang/ro-RO.rc (hunk 1 above)
Comment (AndreiMiloiu) [reply base/applications/notepad/lang/ro-RO.rc]: I changed it back. Now it's just one character.

Comment (AndreiMiloiu) [reply base/applications/notepad/lang/ro-RO.rc]: I understand. I changed it back, so now it's only one character.

File: base/applications/notepad/lang/ro-RO.rc (hunk 2 above)
Comment (AndreiMiloiu) [reply base/applications/notepad/lang/ro-RO.rc]: I fixed it. I didn't understand the rules very well.

Comment (JoachimHenze) [reply base/applications/notepad/lang/ro-RO.rc]: Can happen. That's what a review is for. Resolved.

**Comment by JoachimHenze** [comment]
I do now express again my approval for the current state of this PR, and will merge it in a few days unless anybody else would bring up another issue that I might have overlooked. Well done now from a non-native-romanian-speaker-reviewers point of view!

File: base/applications/notepad/lang/ro-RO.rc [hunk 4]
```diff
@@ -133,21 +133,21 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_CAPTION | WS_SYSMENU
 FONT 8, "MS Shell Dlg"
 CAPTION "Salt la poziție"
 BEGIN
-    LTEXT "Număr linie:", 0x155, 5, 12, 41, 12
+    LTEXT "Număr de linie:", 0x155, 5, 12, 41, 12
```
Comment (JoachimHenze) [inline base/applications/notepad/lang/ro-RO.rc:136]: width 41 might be a bit too narrow to make the longer text fit, but that is just my gutfeeling. I am not sure about that. A preview with ResourceHacker should bring you clarity. I will not withdraw my approval, because I am not certain in this case.

Comment (AndreiMiloiu) [reply base/applications/notepad/lang/ro-RO.rc:136]: This is XP's DLL version.

Comment (JoachimHenze) [reply base/applications/notepad/lang/ro-RO.rc:136]: Yes, but as you can see: XP uses width 47 and not width 41. So that doesn't prove anything. You can build the ros version also with RosBE and drag it into ResourceHacker to be able to preview. I still believe this line might be problematic.
